src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGeneratorOptions.cs

```diff
@@ -61,6 +63,10 @@ public SwaggerGeneratorOptions()
 
         public IList<IDocumentFilter> DocumentFilters { get; set; }
 
```

Comment:
As per my comment below - I'd prefer not accept the type that's also being returned. To be more consistent with the other "*Selector" properties in this class, could we change this signature to the following and update the default selector accordingly?
```csharp
public Func<IEnumerable<AuthenticationScheme>, Dictionary<string, OpenApiSecurityScheme>>
```

Reply:
I've updated this and moved the logic for favoring schemes provided explicitly via options over those produced by the selector in the case where scheme names collide.